WebContent/zip-ext.js

```diff
@@ -208,11 +208,165 @@
 	FileWriter.prototype = new Writer();
 	FileWriter.prototype.constructor = FileWriter;
 
+	function DBWriter(contentType) {
+		var db, tempDB, that = this, blobs, dbName = "zipjs", instance;
```

Comment:
`db` / `tempDB` - Please replace them with less generic variable names and/or add a comment that explains how it works. Synchronization logic is easy to get wrong, and if you document how it is supposed to work, debugging any further bugs will become tremendously easier.